START.md

```diff
@@ -221,6 +223,18 @@ Usage of ./autospotting:
         or if you want to set your bid price to be higher than the on demand
         price to reduce the chances that your spot instances will be terminated.
         (default 1)
+  -spot_price_multiplier float
+        Multiplier for the spot price. Placing bids a fraction of a cent over the
+        current price would aggressively optimize for cost savings since the last
+        runtime hour before spot termination is free of charge.
+        (default 1.1)
```

Comment:
The 1.0 will place spot instance bids at exactly the current market price, the 1.1 is there to offer a 10% buffer above the current price.

But I also find it a bit confusing, perhaps we should switch to percentages and also support absolute value increments, something like spot_price_buffer_percentage and spot_price_buffer_amount, what do you guys think?

Reply:
@kartik894, @xlr-8 what do you think about my proposal to use percentage or absolut values for the spot price buffer?

I think something like spot_price_buffer_percentage=10(and make it a default when the aggressive policy is used) is more intuitive than the current spot_price_multiplier=1.1 for setting the bid to 10% over the current spot price. Also, negative percentage numbers don't make sense in this context, but the multiplier gives the users the chance to mess it up by giving 0.9, which will fail to launch any instances.

This is also quite consistent to the existing percentage/value concept which we use for keeping on-demand capacity, where we can have an absolute value as well as a percentage. This would bid X+current_price, where X may be given as a small fractional number, such as spot_price_buffer=0.001.

Reply:
Your various comments made me think of: https://github.com/cristim/autospotting/pull/175/files#r156044213

We need to check the value anyway, just like it's done for the %/raw-value for on-demand instances. Also this value can't be set at an autoscaling-level if I'm not mistaken. Don't know if that's a goal, but if so some code need to be adapted